editor/assets/chunks/shading-standard-base.chunk

```diff
@@ -50,12 +50,14 @@ struct StandardSurface {
   float roughness;
   float metallic;
   float occlusion;
+  float SpecularIntensity;
 };
 
 vec4 CCStandardShadingBase (StandardSurface s, vec4 shadowPos) {
   // Calculate diffuse & specular
   vec3 diffuse = s.albedo.rgb * (1.0 - s.metallic);
-  vec3 specular = mix(vec3(0.04), s.albedo.rgb, s.metallic);
+  // 0.04 == 0.08*tex.a(0-1)*value(0-1)   valuedefault=0.5
```

Comment:
Do not add this kind of comment, all our code is visible to developers, try to be serious about it